android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -1419,11 +1421,6 @@ public void setRawSrc(@NonNull final Uri uri, @Nullable final String extension)
         }
     }
 
-    public void setTextTracks(ReadableArray textTracks) {
-        this.textTracks = textTracks;
-        initializePlayer(true);
-    }
-
```

Comment:
Don't we want to allow text tracks to be set after the source has been set anymore?

Reply:
There is no current use case for the text tracks to be set after the player has initialised.

The thing I'm trying to avoid is re-initialising the player after playback has already started just to load the text tracks. The playback-related variables should always be set at the same time to avoid this caveat as it has a negative impact on both overall player performance and user experience.